storybook/src/pages/amsterdam/FormPage/FormPage.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Alert,
+  Breadcrumb,
+  Button,
+  Column,
+  Field,
+  FieldSet,
+  FormFieldCharacterCounter,
+  Grid,
+  Heading,
+  Label,
+  Paragraph,
+  Radio,
+  Row,
+  Screen,
+  Select,
+  SkipLink,
+  TextArea,
+  TextInput,
+} from '@amsterdam/design-system-react'
+import { useState } from 'react'
+import { AppHeader } from '../common/AppHeader'
+
+export const FormPage = () => {
+  const [textareaLength, setTextareaLength] = useState(0)
+
+  return (
+    <>
+      <SkipLink href="#main">Direct naar inhoud</SkipLink>
+      <Screen maxWidth="wide">
+        <AppHeader />
+        <Grid paddingTop="small" paddingBottom="medium">
+          <Grid.Cell span={{ narrow: 4, medium: 6, wide: 8 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
+            <Breadcrumb>
+              <Breadcrumb.Link>Home</Breadcrumb.Link>
+            </Breadcrumb>
+            <form className="ams-gap--md" id="main" onSubmit={(e) => e.preventDefault()}>
+              <Heading>Contact</Heading>
```

Comment:
[NLDS recommends placing information about where the user is outside of the `<form>` tag](https://nldesignsystem.nl/richtlijnen/formulieren/meerdere-stappen#plaats-de-informatie-over-waar-de-gebruiker-is-in-de-stappen-boven-het-formulier)